homeassistant/components/nina/quality_scale.yaml

```
@@ -0,0 +1,111 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage:
+    status: done
```

```suggestion
    status: todo
```
`Flow tests should end in ABORT or CREATE_ENTRY` is a blocker for not reaching the goal imo